lib/utils/ember.js

```diff
@@ -222,7 +222,7 @@ function isEmberCoreModule(context, node, moduleName) {
   if (types.isCallExpression(node)) {
     // "classic" class pattern
     return isClassicEmberCoreModule(node, moduleName, context.getFilename());
-  } else if (types.isClassDeclaration(node)) {
+  } else if (types.isClassDeclaration(node) || types.isClassExpression(node)) {
```

Comment:
Class Expressions can be modules since functions can return classes that extend from Ember module types.